app/javascript/react/components/RegionShowContainer.js

```diff
@@ -0,0 +1,33 @@
+import React, { useState, useEffect } from "react";
+import FlavorTile from "./FlavorTile";
+
+const RegionShowContainer = (props) => {
+  const [getFlavors, setFlavors] = useState([]);
+  useEffect(() => {
```

Comment:
add a line between 5 & 6 for readability

Reply:
Added line.